Ontology/Asset/Equipment/LightingEquipment/Luminaire.json

```diff
@@ -2,6 +2,15 @@
   "@id": "dtmi:digitaltwins:rec_3_3:asset:Luminaire;1",
   "@type": "Interface",
   "contents": [
+    {
+      "@type": "Property",
+      "displayName": {
+        "en": "power input"
+      },
+      "name": "powerInput",
+      "schema": "double",
```

Comment:
Watt schema?

Reply:
Sure, I haven't seen where it would be another power unit such as kilowatts